aiohttp_session/redis_storage.py

```diff
@@ -1,4 +1,4 @@
-import asyncio
+import aioredis
```

Comment:
`aioredis` is an **optional** dependency.

Please use the following:
```
try:
     import aioredis
except ImportError:
    aioredis = None
```

In `RedisStorage.__init__` add lines:
```
if aioredis is None:
    raise RuntimeError("Please install aioredis")
```
Also I suggest to add aioredis version check because `aioredis` is an optional dependency and the version cannot be checked by `pip`:
```
from distutils.version import StrictVersion  # aioredis supports strict version schema
if StrictVersion(aioredis.__version__).version < (1, 0):
    raise RuntimeError("aioredis<1.0 is not supported")
```
This explicitly points *user* to the problem.